app/src/forms/form/service.js

```diff
@@ -61,6 +67,9 @@ const service = {
       obj.enableStatusUpdates = data.enableStatusUpdates;
       obj.enableSubmitterDraft = data.enableSubmitterDraft;
       obj.createdBy = currentUser.usernameIdp;
+      obj.schedule = data.schedule;
+      obj.reminder_enabled = data.reminder_enabled ;
```

Comment:
There's a random space at the end before the semi colon.

Reply:
Space remove